apis/extension/resource.go

```diff
@@ -28,6 +28,12 @@ const (
 	BatchCPU    corev1.ResourceName = DomainPrefix + "batch-cpu"
 	BatchMemory corev1.ResourceName = DomainPrefix + "batch-memory"
 
+	GPUCore     corev1.ResourceName = DomainPrefix + "gpu-core"
+	GPUMem      corev1.ResourceName = DomainPrefix + "gpu-memory"
+	GPUMemRatio corev1.ResourceName = DomainPrefix + "gpu-memory-ratio"
```

Comment:
`GPUMem` or `GPUMemory`
`GPUMemRatio` or `GPUMemoryRatio`
Please let me know what you guys think.

Reply:
`GPUMemory` And `GPUMemoryRatio` +1

Reply:
![image](https://user-images.githubusercontent.com/24452340/179680053-b8d7973c-a204-47b0-8929-4a3bd6bed3a7.png)